options.c

```diff
@@ -149,6 +152,20 @@ void parse_options(int argc, char **argv)
 					error(EXIT_FAILURE, 0, "Invalid argument for option -s: %s", optarg);
 				_options.scalemode = s - scalemodes;
 				break;
+			case 'T':
+				if (*optarg == ';') {
+					_options.title_prefix = ++optarg;
+				} else {
+					if ((s = strrchr(optarg, ';')) != NULL) {
+						s[0] = '\0';
+						n = strtol(++s, &end, 0);
```

Comment:
We may wanna check if there was anything after `;` before incrementing the pointer.

Reply:
It's guaranteed to be null-terminated, so if there is nothing there, it'll construct an empty string.